### DISCUSSION TITLE
Copilot overrides import suggestions.

### DISCUSSION THREAD

**Discussion by JoshFerge** [community/community#67979] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Even with copilot turned off, Copilot suggestions still show up for me ahead of import autocomplete. How do i turn this off?
<img width="800" alt="Screenshot 2023-09-22 at 11 00 35 AM" src="https://github.com/community/community/assets/1976777/ee4f381a-ebc2-40e3-a7a9-bced729fb8f8">

**Comment by JoshFerge** [comment]
ah: this setting fixes. 
<img width="490" alt="Screenshot 2023-09-22 at 11 02 51 AM" src="https://github.com/community/community/assets/1976777/d61e3c9c-52e6-44f3-9418-9b690831e900">

Would be good to set precedence to have import suggestions be ahead of copilot

**Reply by abdullzz** [reply]
refund it quick! use gpt4 and codegpt instead. unless you really want to give microsoft some more money to rob!
